doc/nrf/known_issues.rst

```diff
@@ -0,0 +1,572 @@
+     <option value="v130">v1.3.0</option>
+     <option value="v120">v1.2.0</option>
+     <option value="v110">v1.1.0</option>
+     <option value="v100">v1.0.0</option>
+     <option value="v040">v0.4.0</option>
+     <option value="v030">v0.3.0</option>
+   </select>
+
+
+.. HOWTO
+
+   When adding a new version, add it to the dropdown list above.
+
+   When updating this file, add entries in the following format:
+
+   .. rst-class:: vXXX VYYY
+
+   JIRA-XXXX: Title of the issue
+     Description of the issue.
+     Start every sentence on a new line.
+
+     There can be several paragraphs, but they must be indented correctly.
+
+     **Workaround:** The last paragraph contains the workaround.
+
+Common
+******
+
+Crypto
+======
+
+.. rst-class:: v130
+
+NCSDK-5546: Oberon missing symbols for HKDF
+  nRF Oberon v3.0.5 is missing symbols for HKDF using SHA1, which will be fixed in an upcoming version of the library.
+
+  **Workaround:** Use a different backend (for example, vanilla mbed TLS) for HKDF/HMAC using SHA1.
+
+.. rst-class:: v130
+
+Limited support for Nordic Security Module
+  The :ref:`nrfxlib:nrf_security` is currently only fully supported on nRF52840 and nRF9160 devices.
+  It gives compile errors on nRF52832, nRF52833, nRF52820, nRF52811, and nRF52810.
+
+  **Workaround:** To fix the errors, cherry-pick commits in `nrfxlib PR #205 <https://github.com/nrfconnect/sdk-nrfxlib/pull/205>`_.
+
+.. rst-class:: v040 v100
+
+Glue layer symbol renaming issue
+  The :ref:`nrfxlib:nrf_security` glue layer is broken because symbol renaming is not handled correctly.
+  Therefore, the behavior is undefined when selecting multiple back-ends for the same algorithm (for example, AES).
+
+Samples and applications
+========================
+
+.. rst-class:: v130
+
+Build configuration issues
+  The build configuration consisting of :ref:`bootloader`, :ref:`secure_partition_manager`, and application does not work.
+
+  **Workaround:** Either include MCUboot in the build or use MCUboot instead of the immutable bootloader.
+
+.. rst-class:: v130
+
+Flash commands only program one core
+  ``west flash`` and ``ninja flash`` only program one core, even if multiple cores are included in the build.
+
+  **Workaround:** Execute the flash command from inside the build directory of the child image that is placed on the other core (for example, :file:`build/hci_rpmsg`).
+
+Secure Partition Manager (SPM)
+==============================
+
+.. rst-class:: v130
+
+NCSIDB-114: Default logging causes crash
+  Enabling default logging in the :ref:`secure_partition_manager` sample makes it crash if the sample logs any data after the application has booted (for example, during a SecureFault, or in a secure service).
+  At that point, RTC1 and UARTE0 are non-secure.
+
+  **Workaround:** Do not enable logging and add a breakpoint in the fault handling, or try a different logging backend.
+
+nRF9160
+*******
+
+Asset tracker
+=============
+
+.. rst-class:: v131
```

Comment:
This probably applies to more releases.

Reply:
Checking with @umapraseeda .

Reply:
Info from Uma: add v130 tag as well, but no earlier ones.